### PR TITLE
[TS-592]<fix>: doc taosdemo BOM error

### PR DESCRIPTION
**Pull request by afwerar**

### CODE REVIEW

File: documentation20/cn/02.getting-started/02.taosdemo/docs.md [hunk 1]
```diff
@@ -442,7 +442,7 @@ TDengine是涛思数据专为物联网、车联网、工业互联网、IT运维
 taosdemo支持两种配置参数的模式，一种是命令行参数，一种是使用json格式的配置文件。
 一、命令行参数
 
--f：指定taosdemo所需参数的meta文件。当使用该参数时，其他所有命令行参数都失效。可选项，缺省是NULL。
+-f：指定taosdemo所需参数的meta文件。当使用该参数时，其他所有命令行参数都失效。可选项，缺省是NULL。建议UTF-8而不是带BOM的UTF-8。
```
Comment (sangshuduo) [inline documentation20/cn/02.getting-started/02.taosdemo/docs.md]: 目前仅支持不含 BOM（byte-order mark）的标准 UTF-8 编码文件。

**Review by stephenkgu — APPROVED** [review]

File: documentation20/cn/02.getting-started/02.taosdemo/docs.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ insert delay, avg:       8.31ms, max:     860.12ms, min:       2.00ms
 ```
 $ taosdemo --help
 
--f, --file=FILE The meta file to the execution procedure.
+-f, --file=FILE The meta file to the execution procedure. Recommend UTF-8 instead of UTF-8 with BOM.
```
Comment (sangshuduo) [inline documentation20/cn/02.getting-started/02.taosdemo/docs.md]: Currently, we support standard UTF-8 (without BOM) encoded files only.

**Review by sangshuduo — CHANGES_REQUESTED** [review]

File: documentation20/cn/02.getting-started/02.taosdemo/docs.md (hunk 1 above)
Comment (afwerar) [reply documentation20/cn/02.getting-started/02.taosdemo/docs.md]: ok

File: documentation20/cn/02.getting-started/02.taosdemo/docs.md (hunk 2 above)
Comment (afwerar) [reply documentation20/cn/02.getting-started/02.taosdemo/docs.md]: ok

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**